internal/saucecloud/cloud.go

```diff
@@ -336,6 +340,13 @@ func (r *CloudRunner) runJobs(jobOpts chan job.StartOptions, results chan<- resu
 			continue
 		}
 
+		if opts.Attempt+1 < opts.MaxAttempt && opts.PassCount < opts.MinPass {
```

Comment:
Wouldn't that be more readable ?
```suggestion
		if opts.Attempt == opts.MaxAttempt && opts.PassCount < opts.MinPass {
```